app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,5 @@
+class FoodsController < ApplicationController
+  def index
+    @facade = FoodsFacade.new(params)
```

Comment:
Sending the entire params object down means that either the facade or service has to figure out which param is required, I think it might be better to just send `params[:q]` here and let the controllers handle which params.